app/src/main/java/de/danoeh/antennapod/preferences/PreferenceUpgrader.java

```diff
@@ -117,5 +120,15 @@ private static void upgrade(int oldVersion, Context context) {
                 prefs.edit().putString(UserPreferences.PREF_DRAWER_FEED_COUNTER, "2").apply();
             }
         }
+        if (oldVersion < 2070095) {
+
+            long value = Long.parseLong(SleepTimerPreferences.lastTimerValue());
+            TimeUnit unit = SleepTimerPreferences.UNITS[SleepTimerPreferences.lastTimerTimeUnit()];
+
+            SleepTimerPreferences.setLastTimer(
+                    String.valueOf(unit.toMinutes(value))
+            );
```

Comment:
No line breaks needed

```suggestion
            SleepTimerPreferences.setLastTimer(String.valueOf(unit.toMinutes(value)));
```